packages/api-reference/src/helpers/getExampleCode.ts

```diff
@@ -32,6 +32,11 @@ export async function getExampleCode(
     )
   }
 
+  // Prevent snippet generation if starting by a variable
```

Comment:
oh is this because the variables are not replaced?

Reply:
yeah, it throws an error when an url doesn’t have a valid protocol